packages/core/src/indices/list/ListIndex.ts

```diff
@@ -30,6 +30,7 @@ export enum ListRetrieverMode {
 export interface ListIndexOptions {
   nodes?: BaseNode[];
   indexStruct?: IndexList;
+  indexID?: string;
```

Comment:
Small thing, but we camelcase this as indexId in other parts of the code. There's no incorrect option: i've written both Id and ID in code before and I've seen it both ways in APIs, but since we're using Id in the rest of the code base let's keep it as Id.